src/modules/75-cf/pages/feature-flags/components/StaleFlagActions/__tests__/StaleFlagActions.test.tsx

```diff
@@ -42,4 +43,48 @@ describe('StaleFlagActions', () => {
     expect(screen.queryByRole('button', { name: 'cf.staleFlagAction.readyForCleanup' })).not.toBeInTheDocument()
     expect(screen.queryByRole('button', { name: 'cf.staleFlagAction.learnMore' })).not.toBeInTheDocument()
   })
+
+  test('it should render the info drawer when info button is clicked', async () => {
+    renderComponent()
+
+    userEvent.click(screen.getByRole('button', { name: 'cf.staleFlagAction.learnMore' }))
+    await waitFor(() => {
+      expect(screen.getByRole('button', { name: 'close' })).toBeVisible()
+      expect(screen.getByText('cf.staleFlagAction.flagLifecycleExplained')).toBeVisible()
+      expect(screen.getByText('cf.staleFlagAction.flagLifecycleDesc')).toBeVisible()
+      expect(screen.getByRole('button', { name: 'common.gotIt' })).toBeVisible()
+    })
+  })
+
+  test('it should close the info drawer when the x button is clicked', async () => {
+    renderComponent()
+
+    userEvent.click(screen.getByRole('button', { name: 'cf.staleFlagAction.learnMore' }))
+
+    await waitFor(() => {
+      expect(screen.getByRole('button', { name: 'close' })).toBeVisible()
+    })
+
+    userEvent.click(screen.getByRole('button', { name: 'close' }))
+
+    await waitFor(() => {
+      expect(screen.queryByRole('button', { name: 'close' })).not.toBeInTheDocument()
+    })
+  })
+
+  test('it should close the info drawer when the got it button is clicked', async () => {
+    renderComponent()
+
+    userEvent.click(screen.getByRole('button', { name: 'cf.staleFlagAction.learnMore' }))
+
+    await waitFor(() => {
+      expect(screen.getByRole('button', { name: 'common.gotIt' })).toBeVisible()
+    })
+
+    userEvent.click(screen.getByRole('button', { name: 'common.gotIt' }))
+
+    await waitFor(() => {
+      expect(screen.queryByRole('button', { name: 'common.gotIt' })).not.toBeInTheDocument()
+    })
```

Comment:
Not a problem so just FYI, but we're essentially doing the same check 3 times here. You could assign to a variable and make things a _wee_ bit faster.
```suggestion
    const gotItBtn = await screen.findByRole('button', { name: 'common.gotIt' })
    
    expect(gotItBtn).toBeVisible()

    userEvent.click(gotItBtn)

    await waitFor(() => expect(gotItBtn).toBeInTheDocument())
```

Reply:
Thanks have implemented this, the last line however needs to be queryBy so I have left as is for that